config/redirects.js

```diff
@@ -2236,6 +2236,7 @@ module.exports = [
   { from: '/rules/current/redirect', to: '/rules/guides/redirect' },
   { from: '/rules/current/metadata-in-rules', to: '/rules/guides/metadata' },
   { from: '/rules/current/csharp', to: '/rules/guides/csharp' },
+  { from: '/rules/guides/csharp', to: '/rules' },
```

Comment:
Awesome, didn't know we had this capability! 👍

Reply:
And I assume the redirect rule above this line will chain through to this one? So `'/rules/current/csharp'` will end up at `'/rules'`?

Reply:
Yes. Although, I guess technically I could just make it go directly to /rules without the jump.

Reply:
In fact, that's a better choice. Fixed it.